_includes/footer.html

```diff
@@ -0,0 +1,3 @@
+<footer>
+  <span>&copy; 2011-2015 TJ Holowaychuk, Travis Jeffery, Christopher Hiller & Joshua Boy Nicolai Appelman. All rights reserved.  Licensed <a href="http://opensource.org/licenses/MIT" target="_blank">MIT</a></span>
```

Comment:
this is a bit ambiguous.  but at any rate, I don't think it's appropriate to release this code under the MIT license.

I believe because it's under MIT, and others besides TJ would own the copyright to this particular incarnation, we're allowed to relicense the site.

As such, a more suitable license would be [CC](http://creativecommons.org).  Let's change this text to the following, if you don't mind:

```
<a rel="license" href="http://creativecommons.org/licenses/by-sa/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by-sa/4.0/80x15.png" /></a><br /><span xmlns:dct="http://purl.org/dc/terms/" href="http://purl.org/dc/dcmitype/InteractiveResource" property="dct:title" rel="dct:type">mochajs.org</span> is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by-sa/4.0/">Creative Commons Attribution-ShareAlike 4.0 International License</a>.<br />Based on a work at <a xmlns:dct="http://purl.org/dc/terms/" href="https://github.com/mochajs/mocha/tree/gh-pages" rel="dct:source">https://github.com/mochajs/mocha/tree/gh-pages</a>.
```

Then we may want to update `README.md` to note that while the *source* is licensed MIT, a portion of the contents of the `_site/` directory are in fact licensed with CC.